.github/CODEOWNERS

```diff
@@ -1436,23 +1454,18 @@ sdk/dnsresolver/ci.mgmt.yml @qiaozha @dw511214992 @MaryGao
 ###########
 # Eng Sys
 ###########
-/eng/   @ckairen @mikeharder @weshaggard @benbp
-/**/tests.yml   @ckairen @benbp
-/**/ci.yml      @ckairen @benbp
+/eng/           @ckairen @mikeharder @weshaggard @benbp
 
 ###########
 # Config
 ###########
 /.vscode/ @mikeharder @ckairen @deyaaeldeen
 /common/ @mikeharder @ckairen @witemple-msft @deyaaeldeen
 /common/config/rush/pnpm-lock.yaml @ckairen @witemple-msft @deyaaeldeen @jeremymeng
-/common/smoke-test @mikeharder @ckairen @witemple-msft @deyaaeldeen @benbp
+/common/smoke-test/ @mikeharder @ckairen @witemple-msft @deyaaeldeen @benbp
 
 /rush.json @mikeharder @ckairen
 /tsconfig.json @mikeharder @ckairen @witemple-msft @deyaaeldeen
 /**/tsconfig.json @witemple-msft @deyaaeldeen
 /**/tsconfig.strict.json @deyaaeldeen
 /.scripts/ @mikeharder @ckairen
-
-# Add owners for notifications for specific pipelines
-/eng/pipelines/aggregate-reports.yml         @xirzec @jeremymeng
```

Comment:
@konrad-jamrozik I think you forgot to move this back down and that means that this will now show as being owned by /eng/ as the codeowner finds the latest match in the file so the more specific matches should generally be later in the file.

Reply:
Ungh! Thanks for catching it!

Reply:
@weshaggard 
- https://github.com/Azure/azure-sdk-for-js/pull/24841